games-emulation/pin8/pin8-1.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A Chip-8 emulator"
+DESCRIPTION="This is a Chip-8 emulator. Hence, it allows you to run Chip-8 \
+games  and software. Blitz, Blinky and Tetris are some games available for \
+this platform. A sample game, a flappy bird clone, is also available in the \
+webpage of this program. \Pin8 features responsive design, real keyboard \
+compatibility, sound and two running modes."
+HOMEPAGE="https://github.com/cnelmortimer/Pin8/blob/master/README.md"
+COPYRIGHT="2017 N.C. Cruz"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="5784af55e8e90d658533cce98492d3d13577460f"
+SOURCE_URI="https://github.com/cnelmortimer/Pin8/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="4997ae2d06a2346c37a8e9c9831e0d56f6132685b95054450d7f4c72e2ca6e00"
+SOURCE_DIR="Pin8-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86_64"
+
+PROVIDES="
+	pin8 = $portVersion
+	app:pin8 = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	cmd:g++
+	cmd:make
+	cmd:xres
```

Comment:
cmd:xres is provided by haiku package, I think it can be dropped here.

Reply:
in fact it is a good idea to declare it. If we ever decide to split the haiku package in smaller parts, or move the command to another package, we don't need to update any recipes then. That's the point of all these cmd: and lib: provides, and I see no reason to make an exception for the haiku package.

Reply:
Noted